web/src/app/escalation-policies/PolicyCreateDialog.tsx

```diff
@@ -14,35 +14,29 @@ const mutation = gql`
 `
 
 function PolicyCreateDialog(props: { onClose: () => void }): JSX.Element {
-  const defaultValue = {
+  const [value, setValue] = useState<PolicyFormValue>({
     name: '',
     description: '',
     repeat: { label: '3', value: '3' },
     favorite: true,
-  }
-  const [value, setValue] = useState<PolicyFormValue>(defaultValue)
-  const [createPolicy, createPolicyStatus] = useMutation(mutation, {
+  })
+  const [createPolicy, { loading, data, error }] = useMutation(mutation, {
     variables: {
       input: {
-        name: value && value.name,
-        description: value && value.description,
-        repeat: value && value.repeat.value,
+        name: value.name,
+        description: value.description,
+        repeat: value.repeat?.value ?? 3,
```

Comment:
We already set a default above, and we know the value cannot be null.

```suggestion
        repeat: value.repeat.value,
```

Reply:
This was causing failures until I added that but I can give it another look.